src/pages/conference/timeline/_schedule.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import styled from 'styled-components'
+import ArrowRight from './images/downwards_right.svg'
+import ArrowLeft from './images/downwards_left.svg'
+import { Header } from 'components/elements'
+import device from 'themes/device'
+
+type ScheduleProps = {
+    time?: string | number
+    id?: string
+    name?: string
+    topic?: string
+}
+
+type DataProps = {
+    timings: ScheduleProps[]
+    free_time: string | number
+    speakers: ScheduleProps[]
+}
+
+type ScheduleConferenceProps = {
+    id?: string
+    title?: string
+    date?: string
+    period?: string
+    data?: DataProps[]
+}
+
+type ConferenceProps = {
+    item: ScheduleConferenceProps
+    key: string
+}
+
+const MainContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`
+
+const Container = styled.div`
+    padding-bottom: 80px;
+`
+
+const HeaderTitle = styled.div`
+    padding-bottom: 80px;
+`
+
+const WrapperContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    max-width: 995px;
+`
+
+const ImageContainer = styled.div`
+    @media ${device.tablet} {
+        margin-left: 40px;
+    }
+`
+
+const TimeWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h2:last-child {
+        margin-bottom: 0;
+    }
+
+    @media ${device.tablet} {
+        margin-left: 40px;
+    }
+`
+
+const ContentWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h2:last-child {
+        margin-bottom: 0;
+    }
+`
+
+const ScheduleConference = ({ item }: ConferenceProps) => {
+    const { id, date, data, period, title } = item
+
+    return (
+        <>
+            <HeaderTitle key={id}>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
```

Comment:
Let's create separate styled component with repeatable properties such:
`
font-weight: normal;
text-align: center;
`
And use him in whole component where we have the same properties

Reply:
okay created a reusable component